.github/workflows/api-release.yml

```diff
@@ -6,7 +6,7 @@ on:
 name: API Release
 
 env:
-  GO_VERSION: "1.24.1"
+  GO_VERSION: "1.24.2"
```

Comment:
This probably also should be 1.23.8 (?), although I guess API doesn't produce binaries

Reply:
> although I guess API doesn't produce binaries
Yes . Thats why I didnt change that. But can update to 1.23.8

Reply:
Done.